src/polymer-cli.ts

```diff
@@ -12,6 +12,9 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
+// Bail out if invalid node version
+require('please-upgrade-node')(require('../package.json'));
```

Comment:
We already have similar logic here: https://github.com/Polymer/polymer-cli/blob/master/bin/polymer.js#L27

Also, I'd be hesitant to put statements before import declarations, as that's not actually spec-compliant.